deployments/ansible/roles/collector/README.md

```diff
@@ -4,6 +4,16 @@ Ansible role that installs Splunk OpenTelemetry Connector configured to
 collect metrics, traces and logs from Linux machines and send data to [Splunk 
 Observability Cloud](https://www.splunk.com/en_us/observability.html). 
 
+## Windows
+Currently, the following Windows versions are supported:
+Ansible requires PowerShell 3.0 or newer and atleast .NET4.0 to be installed on Windows host.
+A WinRM listner should be created and activeted. 
```

Comment:
Not a Windows/Ansible expert -- not sure what this is or how to ensure available

Reply:
For detailed setup of Ansible in windows I have added Ansible docs reference link.